app/docs/setup_code.tsx

```diff
@@ -175,126 +199,163 @@ export default class SetupCodeComponent extends React.Component {
     event.target.innerText = "Copied!";
   }
 
+  onChangeCredential(e: React.ChangeEvent<HTMLSelectElement>) {
+    this.setState({ selectedCredentialIndex: Number(e.target.value) });
+  }
+
+  onClickManageKeys(e: React.MouseEvent<HTMLAnchorElement>) {
+    e.preventDefault();
+    router.navigateToSettings();
+  }
+
   render() {
-    if (!this.getResponse()) {
+    if (!this.state.bazelConfigResponse) {
       return <div className="loading"></div>;
     }
+    const selectedCredential = this.getSelectedCredential();
     return (
       <div className="setup">
         <div className="setup-controls">
-          <span className="group-title">Select options</span>
           {this.isAuthEnabled() && (
-            <span className="group">
-              {capabilities.anonymous && (
-                <input
-                  id="auth-none"
-                  checked={this.state.auth == "none"}
-                  onChange={this.handleInputChange.bind(this)}
-                  value="none"
-                  name="auth"
-                  type="radio"
-                />
-              )}
-              {capabilities.anonymous && <label htmlFor="auth-none">No auth</label>}
-              <input
-                id="auth-key"
-                checked={this.state.auth == "key"}
-                onChange={this.handleInputChange.bind(this)}
-                value="key"
-                name="auth"
-                type="radio"
-              />
-              <label htmlFor="auth-key">API Key</label>
-              {this.isCertEnabled() && (
-                <span className="group-section">
+            <span className="group-container">
```

Comment:
make sure to test this logic in 
- open source
- internal with auth enabled, logged in
- internal with auth enabled, anonymous
- internal with auth disabled

Reply:
thanks for doing all of that thorough testing!